src/mapping/tests/RadialBasisFctMappingTest.cpp

```diff
@@ -152,15 +122,16 @@ void testDistributed(const TestContext &    context,
                      int                    inGlobalIndexOffset = 0,
                      bool                   meshIsSmaller       = false)
 {
-  int meshDimension  = inMeshSpec.at(0).position.size();
-  int valueDimension = inMeshSpec.at(0).value.size();
+  int valueDimension = inMeshSpec.vertices.at(0).value.size();
 
-  mesh::PtrMesh   inMesh(new mesh::Mesh("InMesh", meshDimension, testing::nextMeshID()));
-  Eigen::VectorXd inValues = getDistributedMesh(context, inMeshSpec, inMesh, inGlobalIndexOffset);
+  inMeshSpec.meshName = mesh::PtrMesh(new mesh::Mesh("InMesh", inMeshSpec.meshDimension, testing::nextMeshID()));
+  getDistributedMesh(context, inMeshSpec, inGlobalIndexOffset);
```

Comment:
This is how I would like the final version to look like:

```suggestion
  MeshPtr inMesh = getDistributedMesh(context, inMeshSpec, inGlobalIndexOffset);
```

So, `getDistributedMesh` builds a `Mesh` and returns in as a `MeshPtr`.